### PR TITLE
Documentation for OTel .NET AutoInstrumentation 1.3.0

### PR DESCRIPTION
**Pull request by Kielek**
Propagate documentation changes for https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/releases/tag/v1.3.0

### CODE REVIEW

**Review by lachmatt — APPROVED** [review]

**Comment by chalin** [comment]
Oh, so sorry, I missed that an @open-telemetry/dotnet-approvers hadn't approved yet! I thought that this only needed docs approval. I'll be more careful next time. Let me know if this needs to be reverted.

**Comment by Kielek** [comment]
It was approved by @pellared from automatic instrumentation sig. I think that it is fine for automatic only content.

**Pull request merged**
